build/Foundation/dll/Foundation.def

```diff
@@ -6,6 +6,7 @@ LIBRARY Foundation
 
         ; Foundation ObjC classes
         _OBJC_CLASS_NSArray DATA
+        _OBJC_CLASS__NSProxyOrderedSetArray DATA
```

Comment:
I agree with Ben. In general I've learned that the compiler / linker teams have lots of smart people that have been working for years on them. Doing things to simply get warnings / errors to go away is rarely the right call as it usually will reveal a bigger underlying issue in the code. Lets figure out what the error was and see if we can understand why it thinks it needs to be exported.

---
In reply to: [64072253](https://github.com/Microsoft/WinObjC/pull/482#discussion_r64072253) [](ancestors = 64072253,64045867)